backend/glific_migration/sync_credentials/run_credentials.py

```
@@ -0,0 +1,29 @@
+from pathlib import Path
+import json
+from glific_migration.sync_credentials.processor import CredentialProcessor
+
+base_dir = Path(__file__).parent.resolve()
+
+
+def main():
+    with open(base_dir / "../config.json", "r") as file:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)